packages/slice-machine/changelog/versions/0.1.1/index.mdx

```diff
@@ -0,0 +1,23 @@
+### We simplified the way you install Slice Machine
+
+Every user can now easily set up Slice Machine in his project using a single command line : `npx @slicemachine/init`
+
+This command lets users do the following :
+
+- Create or connect to a Prismic repo
+- Detect the project framework
+- Install Slice Machine UI as a dev dependency.
+- Create a sm.json file including the Prismic API endpoint to query content and the Slices path
+- Add a `slicemachine` script in the package.json to be able to run the plugin easily
+
+### You can now log in to your repository from the UI 🍾
+
+To avoid doing back and forth between your terminal and the UI. Now when you try to push your data model to Prismic and you are not connected or your connection expired, you will be asked to login with a direct link to do so.
```

Comment:
`going` instead of `doing` maybe? I'm not sure the sentence sounds that natural in English

Reply:
Fixed :)